### ISSUE TITLE
can't debug netcore 3 on debian linux (buster)

### ISSUE THREAD

**Issue by kttary** [OmniSharp/omnisharp-vscode#2747]
## Environment data
`dotnet --info` output:
```
.NET Core SDK (reflecting any global.json):
 Version:   3.0.100-preview-009812
 Commit:    e3abf6e935

Runtime Environment:
 OS Name:     debian
 OS Version:  
 OS Platform: Linux
 RID:         debian-x64
 Base Path:   /usr/share/dotnet/sdk/3.0.100-preview-009812/

Host (useful for support):
  Version: 3.0.0-preview-27122-01
  Commit:  00c5c8bc40

.NET Core SDKs installed:
  3.0.100-preview-009812 [/usr/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.0.0-preview-18579-0056 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.0.0-preview-27122-01 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```
VS Code version:
C# Extension version: 1.17.1

## Steps to reproduce
1. Add breakpoint
1. Start debugging

## Expected  behavior
stop on break

## Actual behavior
Debug adapter process has terminated unexpectedly (exit code: null)

**Comment by gregg-miskelly** [comment]
I will try to setup a machine to see if I can repro this. In the mean time, can you try [enabling logging](https://github.com/OmniSharp/omnisharp-vscode/wiki/Enabling-C%23-debugger-logging) and see if anything useful is printed.

**Comment by gregg-miskelly** [comment]
I was able to successfully setup a Debian buster machine, and I reproduce your problem.

There are two different problems affecting this scenario (at least that I ran into):
1. vsdbg runs on .NET Core 2.2. which requires libssl 1.0.x. I was able to fix this with `sudo apt-get install libssl1.0.2`.
2. https://github.com/OmniSharp/omnisharp-vscode/issues/2734

**Comment by kttary** [comment]
yes, netcore 2.2 running well with libssl 1.0.1 on my machine but netcore 3.0

**Comment by kttary** [comment]
i switched /usr/share/dotnet folder to use dotnet 3.0 and 2.2. 
i want to put them all together but unsuccessfull yet

**Comment by kttary** [comment]
now i'm able to put netcore 2.2 and 3.0 all together. i open netcore 2.2 and i it stop on break point. then i close and open netcore 3.0 project. it is not stop on break point but the page is shown. hope this help. thank you

**Comment by SgtPeppers** [comment]
Im with the same trouble, i can debug net 2.2 but not 3.0

**Comment by gregg-miskelly** [comment]
Debugging on Linux/Mac didn't work in .NET Core 3.0 preview 1. This has been fixed for the last month in master, so if you try something more recent it should work again.
